NEST-113: Create a tutorial recipe and tutorial steps (#2)
Lombiq/Orchard-Walkthroughs: Readme.md

# Lombiq Walkthroughs for Orchard Core

## About

> Orchard Core Walkthroughs module for teaching Orchard Core fundamentals by guiding trough the user in step by step guides.

Reviewer: ```suggestion
Orchard Core Walkthroughs module for teaching Orchard Core fundamentals by guiding through the user in step by step guides.
```

The module uses [shepherd.js](https://shepherdjs.dev/), for the walkthroughs.

> Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects! You can use the setup recipe called _Walkthroughs_ to enable the feature.

Reviewer: ```suggestion
Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects! You can use the setup recipe called _Walkthroughs_ to setup a site using this feature.
```

## Walkthroughs

- Orchard Core Admin Walkthrough: This walkthrough covers key Orchard Core features, such as content management, user roles, and theme selection, and points users to further learning resources.

## Dependencies

This module has the following dependencies:

> - [Lombiq Node.js Extensions](https://gihub.com/Lombiq/NodeJs-Extensions)

Reviewer: ```suggestion
- [Lombiq Node.js Extensions](https://github.com/Lombiq/NodeJs-Extensions)
```
